h/static/scripts/directive/sidebar-tutorial.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 // @ngInject
-function controller(session) {
+function SidebarTutorialController(session) {
   /*jshint validthis:true */
   var vm = this;
 
   vm.showSidebarTutorial = function () {
-    return session.state.preferences.show_sidebar_tutorial;
+    if (session.state.preferences) {
+      if (session.state.preferences.show_sidebar_tutorial === true) {
```

Comment:
In keeping with our current coding conventions, this should just be `if (session.state.preferences.show_sidebar_tutorial)`